### PR TITLE
Fixing CI tests.

### PR DESCRIPTION
**Pull request by Hadatko**
Signed-off-by: Cervenka Dusan <cervenka@acrios.com>

# Pull request

**Choose Correct**

- [X] bug
- [ ] feature

**Describe the pull request**

Currently the shell scripts may failed but CI will not show errors.

**To Reproduce**

Add error to any shell step and run CI.

**Expected behavior**

CI will show failures

**Screenshots**

**Desktop (please complete the following information):**

- OS:
- eRPC Version:

**Steps you didn't forgot to do**

- [X] I checked if other PR isn't solving this issue.
- [X] I read Contribution details and did appropriate actions.
- [X] PR code is tested.
- [X] PR code is formatted.

**Additional context**

### CODE REVIEW

**Comment by Hadatko** [comment]
Hi @MichalPrincNXP,
we had serious issues with testing erpc using CI. Some of them appeared with using scripts. Now it should work as intended but i don't like much solution in conftest.py file. I will try to update it asap.
But until then i would merge this PR if you agree and provide other solution if i will find. What do you think?

**Review by MichalPrincNXP — APPROVED** [review]

**Comment by MichalPrincNXP** [comment]
> Hi @MichalPrincNXP, we had serious issues with testing erpc using CI. Some of them appeared with using scripts. Now it should work as intended but i don't like much solution in conftest.py file. I will try to update it asap. But until then i would merge this PR if you agree and provide other solution if i will find. What do you think?

I agree, thanks.

**Pull request merged**
